JSTests/stress/arith-abs-on-various-types.js

```diff
@@ -77,6 +77,7 @@ function testAllTypesCall() {
                 throw "Failed testAllTypesCall for input " + testCaseInput[0] + " expected " + testCaseInput[1] + " got " + output;
         }
     }
+    // Because DoubleRep can be three-geared
```

Comment:
Nit: can we have these say "Because DoubleRep has three UseKinds we could pick before getting to the generic version"?

Reply:
Changed :)